frontend/public/actions/features.ts

```diff
@@ -248,6 +248,7 @@ const ssarCheckActions = ssarChecks.map(({ flag, resourceAttributes, after }) =>
     spec: { resourceAttributes },
   };
   const fn = (dispatch) => {
+    dispatch(setFlag(flag, undefined));
```

Comment:
I wonder if this will cause unnecessary rerenders on initial console load. Would it be better to move this logic to `startImpersonate` so it only runs on impersonation? Maybe `endImpersonate` too? Then we could add an action to clear SSAR flags.

https://github.com/spadgett/console/blob/4ad6ab2e964579003a817ab718cd56451ce18f81/frontend/public/actions/ui.ts#L193-L195

Reply:
you're probably right since default feature state does not have flag keys in and by calling setFlag(flag, undefined) we create them.

I've created new action `clearSSARFlags` which runs on start and end of impersonation